corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/utils.js

```diff
@@ -85,14 +85,14 @@ hqDefine("cloudcare/js/formplayer/menus/utils", function () {
         FormplayerFrontend.regions.getRegion('breadcrumb').show(breadcrumbView);
     };
 
-    var showFormMenu = function (langs, enableLanguageMenu) {
+    var showFormMenu = function (langs) {
         var langModels,
             langCollection;
 
         FormplayerFrontend.regions.addRegions({
             formMenu: "#form-menu",
         });
-        if (langs && enableLanguageMenu) {
+        if (langs) {
             langModels = _.map(langs, function (lang) {
                 return {
                     lang: lang,
```

Comment:
Is it possible to look up the language names here rather than modifying the rendered HTML later?

Reply:
Made the change - https://github.com/dimagi/commcare-hq/pull/33158/commits/4e13c3c5c1431bf56460480fefdb88c8f728d877